engine/platform/win32/lib_win.c

```diff
@@ -992,15 +991,15 @@ qboolean COM_CheckLibraryDirectDependency( const char *name, const char *depname
 	
 	if( directory->Size <= 0 )
 	{
-		Q_snprintf( errorstring, sizeof( errorstring ), "%s has no dependencies. Is this dll valid?\n" );
+		Q_snprintf( errorstring, sizeof( errorstring ), "%s has no dependencies. Is this dll valid?\n", name );
 		goto libraryerror;
 	}
 
-	importDesc = (PIMAGE_IMPORT_DESCRIPTOR)CALCULATE_ADDRESS( data, directory->VirtualAddress );
+	importDesc = (PIMAGE_IMPORT_DESCRIPTOR)ImageRvaToVa( old_header, data, old_header->OptionalHeader.DataDirectory[IMAGE_DIRECTORY_ENTRY_IMPORT].VirtualAddress, NULL );
 
 	for( ; !IsBadReadPtr( importDesc, sizeof( IMAGE_IMPORT_DESCRIPTOR)) && importDesc->Name; importDesc++ )
 	{
-		const char *importName = ( const char* )CALCULATE_ADDRESS( data, importDesc->Name );
+		const char *importName = ImageRvaToVa( old_header, data, importDesc->Name, NULL );
```

Comment:
Isn't dbghelp is supposed to be optional?

By looking at ReactOS code, CALCULATE_ADDRESS probably isn't correct.
https://github.com/reactos/reactos/blob/177ae91bf63810daae17f15bb21be16592c485a1/sdk/lib/rtl/image.c#L341

Reply:
CALCULATE_ADDRESS would cause random access violations for me
it actually worked sometimes but then i could run the engine without any changes and it wouldn't work without any reason
this function works consistently for me